docgen/src/Getting_started_React_native.md

```diff
@@ -0,0 +1,642 @@
+---
+title: React Native
+layout: main.pug
+category: gettingstarted
+withHeadings: true
+navScroll: true
+---
+
+## Welcome to React InstantSearch
+
+In this tutorial, you’ll learn how to:
+
+* install `react-instantsearch` in your [React Native](https://facebook.github.io/react-native/)  project
+* bootstrap an app using the `<InstantSearch>` component
+* display infinite results from Algolia
+* add widgets to filter the results
+* connect your own component to the search
```

Comment:
I am not sure which component we're talking about here ("connect your own component") since it seems it's not mention anywhere else, could you give an example ? :)